src/js/view/controls/components/menu/menu.js

```diff
@@ -85,6 +85,8 @@ export default class Menu extends Events {
                 case 'Esc':
                     focusEl = settingsElement;
                     this.close(event);
```

Comment:
Is line 87 still necessary?

Reply:
I believe so - as there's other logic in the `close` method. However, it's possible that none of this logic actually has to be invoked if we're simply closing the main menu. What do you think?

https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controls/components/menu/menu.js#L373